src/model/mixins/WithOptions.ts

```diff
@@ -3,20 +3,33 @@ import { Types, Type, Leaf, LeafList } from '@128technology/yinz';
 
 const { EnumerationType, IdentityRefType, DerivedType, BooleanType } = Types;
 
+interface IEnumeration {
```

Comment:
You can make a file in like `src/util/types.ts` where you export this so you don't have to repeat it